dangerfile.js

```diff
@@ -91,3 +91,29 @@ if (capitals.created || underscores.created) {
     '(dash-separated lowercase).',
   ].join(''));
 }
+
+const all_files = danger.git.created_files.concat(danger.git.modified_files);
+
+all_files.forEach(function(file) {
+  danger.git.diffForFile(file).then(function(diff) {
+    if (/\+.*assert[(.]/.test(diff.diff)) {
+      warn([
+        `Found 'assert' statement added in ${file}. `,
+        'Please ensure tests are written using Chai ',
+        '[expect syntax](http://chaijs.com/guide/styles/#expect)'
+      ].join(''));
+    }
+  });
+});
+
+all_files.forEach(function(file) {
+  if (/^services\/.+\/.+\.js$/.test(file) && file.endsWith('.js') && !file.endsWith('.tester.js')) {
```

Comment:
You **_should_** be able to get away with just using this regex:
```regex
if (/^services\/.+\/[^\.]+(?:(?!\.tester))\.js$/.test(file)) {
```
But the way you have it currently is more self explanatory.

Reply:
I think in this case the longer expression is more readable